Tests/BackendTest/BlockChangeRequestTest.cs

```diff
@@ -74,6 +74,9 @@ public async Task ExchangeRequests_ExchangingRequests_ExchangedRequests()
             blockChangeService.FindWaitingStudentRequests(student2.Id).Result.Count.Should().Be(2);
             blockChangeService.FindWaitingStudentRequests(student3.Id).Result.Count.Should().Be(1);
 
+            await blockChangeService.CancelExchangeRequest(blockToChange3);
+            blockChangeService.FindWaitingStudentRequests(student2.Id).Result.Count.Should().Be(1);
+
```

Comment:
For every logic test, we should have separated test, this is test for exchange, create new one test for cancel...